docs/abi-spec.rst

```diff
@@ -537,7 +537,7 @@ For example,
 ::
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >0.6.99 <0.8.0;
+    pragma solidity >=0.7.0;
```

Comment:
Wouldn't `^0.7.0` be the appropriate one?

Reply:
Well, if this snippet doesn't break `0.8.0`, then `>=0.7.0` is more appropriate, isn't it?

Reply:
We don't now yet if it doesn't break with 0.8.0, because it is not out yet. In general I think we tended to use `^0.7.0` so that someone visiting the wrong version of the documentation will not raise a complaint that the example does not work, because it would error out on a later compiler.

Reply:
Ok. I made a PR that changes it https://github.com/ethereum/solidity/pull/9911